src/predict_request_validation_utils.cpp

```diff
@@ -114,9 +114,31 @@ class RequestValidator {
     Status validateNumberOfShapeDimensions(const ovms::TensorInfo& inputInfo, const InputTensorType& proto) const;
     Status validatePrecision(const ovms::TensorInfo& inputInfo, const InputTensorType& proto) const;
     bool checkIfBinaryInputUsed(const InputTensorType& proto, const std::string inputName) const;
+    Status validateRequestCoherency() const;
     Status validate();
 };
 
+template <>
+Status RequestValidator<TFSRequestType, TFSInputTensorType, TFSInputTensorIteratorType, TFSShapeType>::validateRequestCoherency() const {
+    return StatusCode::OK;
```

Comment:
Tensorflow serving has multiple places for content as well. Why do we need to validate it for KFS? If we really need to do so, we should do that for TFS as well.

Reply:
Its specifically mentioned in KFS API
https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L152
https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L200